Sources/Sound.swift

```diff
@@ -275,6 +275,22 @@ open class Sound {
             }
         }
     }
+    
+    /// Set volume with duration i.e. fade to a new volume over a duration
+    @available(iOS 10.0, *)
```

Comment:
this will not compile for macOS and tvOS. the minimum versions should be specified for these platforms as well
```suggestion
    @available(iOS 10.0, tvOS 10.0, macOS 10.12, *)
```